### DISCUSSION TITLE
How to set priority for gotify push notification?

### DISCUSSION THREAD

**Discussion by gabrielnemeth** [containrrr/watchtower#1252] [Q&A] [answered]
Gotify has support for notification priority. Currently, I got no sound on my Android phone when watchtower sends a notification through Gotify. Could I somehow define this priority in my watchtower docker-compose file? I could find only 3 variables in documentation: `WATCHTOWER_NOTIFICATIONS`, `WATCHTOWER_NOTIFICATION_GOTIFY_URL`, `WATCHTOWER_NOTIFICATION_GOTIFY_TOKEN`.

**Comment by piksel** [comment, marked as answer]
It's not supported by the legacy notifications, but if you can switch to shoutrrr URLs by replacing
```
WATCHTOWER_NOTIFICATIONS=gotify
WATCHTOWER_NOTIFICATION_GOTIFY_URL=https://gotify.example.com
WATCHTOWER_NOTIFICATION_GOTIFY_TOKEN=AzyoeNS.D4iJLVa
```
with
```
WATCHTOWER_NOTIFICATIONS=shoutrrr
WATCHTOWER_NOTIFICATION_URL=gotify://gotify.example.com/AzyoeNS.D4iJLVa/?priority=1
```
